### PR TITLE
Update create_image_from_disk() to specify only the instance_name instead of the ful…

### LINKED ISSUE #11111: create_image_from_disk() from "Create Custom Images" section fails with ValueError: Invalid request.
## In which file did you encounter the issue?
https://github.com/GoogleCloudPlatform/python-docs-samples/blob/HEAD/compute/client_library/snippets/images/create.py#L88-L153

## Did you change the file? If so, how?
No - I encountered this issue running it as-is with `google-cloud-compute-1.15.0`

## Describe the issue
Running the create_image_from_disk() function with existing project, zone, and instance names results in the following error:

```
ValueError: Invalid request.
Some of the fields of the request message are either not initialized or initialized with an invalid value.
Please make sure your request matches at least one accepted HTTP binding.
To match a binding the request message must have all the required fields initialized with values matching their patterns as listed below:
```

### PR DESCRIPTION
**Pull request by PaulFenton**
## Description

Fixes #11111 

Updates the invocation of `instance_client.get()` to properly use the instance name of the disk's user rather than the entire path (which causes a ValueError)

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [x] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Comment by rsamborski** [comment]
Thank you @PaulFenton for the contribution. Please sign our CLA, so we can accept it. Also due to the way these samples are constructed the change needs to be made in the proper `ingredient file`.

@m-strzelczyk please advice Paul on the proper file location and on how to regenerate the snippet.

**Comment by PaulFenton** [comment]
ok, CLA is signed.

**Comment by m-strzelczyk** [comment]
@PaulFenton The file you have edited is auto-generated, so please instead of editing this file, could you edit: `compute/client_library/ingredients/images/create.py` ? And then run `compute/client_library/sgs.py` as described in README? That way we will keep everything in sync and well organized.

File: README.md [hunk 1]
```diff
@@ -12,61 +12,60 @@ Check out some of the samples found on this repository on the [Google Cloud Samp
```
Comment (PaulFenton) [inline README.md:12]: Noticed the readme's numbered lists were all "1. , 1. , 1. ..."

File: compute/client_library/snippets/images/create.py
Comment (PaulFenton) [inline compute/client_library/snippets/images/create.py:1]: generated by running `python3 sgs.py generate`

**Comment by PaulFenton** [comment]
@m-strzelczyk 
I updated the ingredient file and ran `python3 sgs.py generate`. Also found a small readme fix.

Good to go?

**Review by m-strzelczyk — DISMISSED** [review]
Please revert the README.md list numbering changes. After that feel free to merge :)

**Review by rsamborski — APPROVED** [review]

**Pull request merged**